### ISSUE TITLE
Bug: Parameter fixer always appends {} to parameter with any type

### ISSUE THREAD

**Issue by orionna319** [JoshuaKGoldberg/TypeStat#1278]
Thank you very much for your contribution. I think this project is very helpful for those who are new to ts. When I use it in my project, I found the following problems.

### 🐛 Bug Report

-   TypeStat version: 0.7.0
-   TypeScript version: x.x.x
-   Node version: 16.20.0

#### Actual Behavior

When i run `npx typestat --config typestat.json`, I get this.
![image](https://github.com/JoshuaKGoldberg/TypeStat/assets/85230052/eee36337-a17f-4160-93a9-3d727abcf9ab)

#### Expected Behavior

```ts
interface FetchData {
  data: any[];
  total: number;
}
export default function useTable(
  fetchData: (params?: any) => Promise<FetchData>,
  options: any = {}
) {}
```

#### Reproduction
https://stackblitz.com/edit/stackblitz-starters-capyrt?file=index.ts

**Comment by JoshuaKGoldberg** [comment]
Aha, thanks for trying out TypeStat @orionna319! Confirmed on the bug - I can send a quick fix now.

**Comment by JoshuaKGoldberg** [comment]
This should be fixed in `0.7.2`. Please do post back if it's not, or other bugs show up!
